src/configurator/__tests__/offline.js

```diff
@@ -0,0 +1,146 @@
+import test from "ava"
+
+import configurator from ".."
+
+test("should default to false for 'offline' option", (t) => {
+  t.is(
+    configurator({}, []).offline,
+    false
+  )
+})
+
+test("should provide default offlineConfig when 'offline' = true", (t) => {
+  t.deepEqual(
+    configurator({ phenomic: { offline: true } }, []).offlineConfig,
+    {
+      serviceWorker: true,
+      appcache: true,
```

Comment:
how browser will behave with appcache + service worker enabled?! Is this not overkill?! Does service worker take precedence?